samples/go/csv/read_test.go

```diff
@@ -133,6 +133,28 @@ g,h,i,j
 `
 	testTrailingHelper(t, dataString)
 }
+func TestEscapeStructFieldFromCSV(t *testing.T) {
```

Comment:
I was using goreturns, but now that I switched to gofmt I didn't see a difference( I also ran it from commandline), do I need any additional linting information ? New to go :)